src/main/kotlin/com/team4099/robot2025/subsystems/arm/Arm.kt

```diff
@@ -0,0 +1,88 @@
+package com.team4099.robot2025.subsystems.arm
+
+import com.team4099.robot2025.subsystems.superstructure.Request
+import com.team4099.robot2025.util.CustomLogger
+import org.littletonrobotics.junction.Logger
+import org.team4099.lib.units.derived.Angle
+import org.team4099.lib.units.derived.ElectricalPotential
+import org.team4099.lib.units.derived.degrees
+import org.team4099.lib.units.derived.volts
+
+class Arm(val io: ArmIO) {
+    val inputs = ArmIO.ArmIOInputs()
+
+
+    var armTargetVoltage: ElectricalPotential = 0.0.volts
+    var armTargetPosition: Angle = 0.0.degrees
+    var armTolerance: Angle = 0.0.degrees
+
+    var currentState = ArmState.UNINITIALIZED
+    var currentRequest: Request.ArmRequest = Request.ArmRequest.Zero()
```

Comment:
somewhat scared about this since cancoders sometimes startup late. but this should be fixed with the absolute stuff